topi/python/topi/cuda/vision.py

```diff
@@ -0,0 +1,99 @@
+# pylint: disable=invalid-name, unused-variable
+"""Schedule for vision operators"""
+from __future__ import absolute_import as _abs
+import tvm
+import topi
+from .. import tag
+from .. import generic
+
+@generic.schedule_reorg.register(["cuda", "gpu"])
+def schedule_reorg(outs):
+    """Schedule for reorg operator.
+
+    Parameters
+    ----------
+    outs: Array of Tensor
+        The computation graph description of reorg
+        in the format of an array of tensors.
+
+    Returns
+    -------
+    s: Schedule
+        The computation schedule for reorg.
+    """
+    return topi.cuda.schedule_injective(outs)
+
+@generic.schedule_shortcut.register(["cuda", "gpu"])
+def schedule_shortcut(outs):
+    """Schedule for shortcut operator.
+
+    Parameters
+    ----------
+    outs: Array of Tensor
+        The computation graph description of shortcut
+        in the format of an array of tensors.
+
+    Returns
+    -------
+    s: Schedule
+        The computation schedule for shortcut.
+    """
+    return topi.cuda.schedule_injective(outs)
+
+@generic.schedule_region.register(["cuda", "gpu"])
+def schedule_region(outs):
+    """Schedule for region operator.
+    Parameters
+    ----------
+    outs: Array of Tensor
+        The computation graph description of region
+        in the format of an array of tensors.
+
+    Returns
+    -------
+    s: Schedule
+        The computation schedule for region.
+    """
+
+    outs = [outs] if isinstance(outs, tvm.tensor.Tensor) else outs
+    s = tvm.create_schedule([x.op for x in outs])
+    output = outs[0].op.output(0)
+    num_thread = 64#tvm.target.current_target(allow_none=False).max_num_threads
```

Comment:
is there any reason for using 64 threads specifically?

Reply:
please remove the commented line and update add a line of comment saying why 64 is used.